api/AudioContext.json

```diff
@@ -121,7 +121,8 @@
               "version_added": "42"
             },
             "safari": {
-              "version_added": null
+              "version_added": null,
```

Comment:
Should be true ;-)

Reply:
Fixed. Thanks, I am not able to test Safari compatibility.